app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -158,25 +157,23 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         // See #1881 for details
```

Comment:
nit: can we add a comment to explain how this onboarding experience is intended to work? i.e. we open onboarding, then return to MainActivity so a new onboarding is shown, then return to MainActivity, etc. so we see all of them because this is in onResume?

fwiw, I do very much prefer this approach to launching onboarding to the previous one.